.github/workflows/push.yml

```
@@ -60,3 +60,29 @@
           fail_ci_if_error: false
           name: '${{env.CODECOV_UNIQUE_NAME}}'
 
+  Deploy-Docusaurus:
+    name: Deploy https://admin-docs.talawa.io website
+    runs-on: ubuntu-latest
+    needs: [Code-Coverage]
+    # Run only if the develop-postgres branch and not dependabot    
+    if: ${{ github.actor != 'dependabot[bot]' && github.event.pull_request.base.ref == 'develop-postgres' }}
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)